### PR TITLE
Db week1

### PR DESCRIPTION
**Pull request by husenjanobul**

### CODE REVIEW

File: Week1/hm/queries4.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+});
+
+connection.connect();
+
+const queries =[
+    // countries with population greater than 8 million
+    'SELECT name FROM country WHERE population>=8000000',
+
+    // countries that have “land” in their names
+    'SELECT name FROM country WHERE name LIKE "%lands"',
```
Comment (roxana-lafuente) [inline Week1/hm/queries4.js]: All queries are OK except this one but I think it's just a typo and you meant:

```
SELECT name FROM country WHERE name LIKE "%land%"
```

:)

File: Week1/hm/queries4.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    // cities with population in between 500,000 and 1 million
+    'SELECT name FROM city WHERE population>=500000 and population<= 1000000',
```
Comment (roxana-lafuente) [inline Week1/hm/queries4.js]: Another option would have been to use between. More info here: https://www.techonthenet.com/mysql/between.php

File: Week1/hm/queries5.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    host: 'localhost',
+    user: 'hyfuser',
+    password : 'hyfpassword',
+    database: 'world'
+});
+
+connection.connect();
+
+const queries =[
+    // cities with population in between 500,000 and 1 million
```
Comment (roxana-lafuente) [inline Week1/hm/queries5.js]: Careful here: the comments don't match what the queries do :)

File: Week1/hm/createDatabase.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password : 'hyfpassword',
+})
+
+connection.connect();
```
Comment (roxana-lafuente) [inline Week1/hm/createDatabase.js]: It's always a good idea to add validation on everything that could go on. And connecting to the DB is one of those places where tons of things can go wrong: user does not exist, wrong password, database does not exist, etc

File: Week1/hm/queries5.js (hunk 3 above)
Comment (husenjanobul) [reply Week1/hm/queries5.js]: Thank you a lot for review and feedback . 
I’ll check again. :)

**Pull request closed without merging**